### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by PatrykMis**
Update Polish translation with additions and some minor spelling corrections.

### CODE REVIEW

**Comment by flackbash** [comment]
Great, thank you and sorry for the late reply.

Karl added support for plural strings in #144 and already fixed the typo you discovered.
I just had to look through it before merging and didn't have time for it for a while, so I only just merged it.

I also changed some string names and contents in 5fa223b48279edbf0f8ac1f9d321237be9a1488a (changed "delete from library" strings into "remove from library" strings for consistency).
Could you update your translation accordingly and remove the "Fix typo" commit?
You can squash it all into one commit and then I'll merge it :)

[The Android Developer Guide](https://developer.android.com/guide/topics/resources/string-resource#Plurals) gives a good introduction to plural strings in case they are new to you.

**Comment by huftis** [comment]
@PatrykMis Regarding plural strings and updating the translation files: If you’re *manually* editing the .xml file, you might want to consider switching to using the android2po tool: https://github.com/miracle2k/android2po

Basically, this generates (exports) standard Gettext .po files, with plural strings properly handled. Then you can use a normal translation editor, like the excellent Lokalize, to edit the translation, and later convert (import) the .po file to the Android string .xml file (again using android2po).

Use `a2po init --groups strings` from the `AudioAnchor/app/src/main` folder to get started. See the documentation for more information.

I use this approach for handling the Norwegian Nynorsk translation, and it works very well. BTW, I find it best to generate (export) the strings to a template file (.pot) and update the .po file by using

```console
msgmerge --update --previous --backup=none my-translation-file.po template-file.pot
```

The `--previous` argument ensures that the .po file also embeds the previous version of any changed English strings. Lokalize will then show a coloured diff view, making it easy to see what‘s changed (and do the corresponding change in the translation, if relevant).

**Comment by PatrykMis** [comment]
@huftis Thank you for your great tips.

@flackbash Updated everything. Commits can be squashed while merging so you are free to do it, my knowledge is too little to do it (properly) myself, sorry.

File: app/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -1,47 +1,113 @@
...
     <string name="empty_title_error">Proszę wpisać tytuł swojej zakładki.</string>
-    <string name="jumped_to_bookmark">Skoczono do zakładki \"%s\" w %s</string>
+    <string name="jumped_to_bookmark">Skoczono do zakładki \"%s\" w %s.</string>
     <string name="bookmark_deleted">Usunięto zakładkę.</string>
-    <string name="tracks_deleted_from_db">Usunięto %d ścieżkę(-ki) dźwiękową(-wych) z bazy danych.</string>
+    <string name="bookmark_added_toast">Dodano zakładkę \"%s\" w %s.</string>
+    <plurals name="tracks_deleted">
+        <item quantity="one">Usunięto %d ścieżkę dźwiękową.</item>
+        <item quantity="few">Usunięto %d ścieżki dźwiękowe.</item>
+        <item quantity="other">Usunięto %d ścieżek dźwiękowych.</item>
```
Comment (flackbash) [inline app/src/main/res/values-pl/strings.xml:31]: I get the error message for this and all other plural strings that for Polish the quantity *many* should also be defined. I don't know any Polish, so I don't know if it makes any difference in this scenario. Was there a reason you didn't add it?

File: app/src/main/res/values-pl/strings.xml [hunk 2]
```diff
@@ -54,24 +120,57 @@
...
+    <string name="untitled_bookmark">Niezatytułowana</string>
+    <string name="delete_from_device">Usuń z urządzenia</string>
     <string name="remove_from_db">Usuń z bazy danych</string>
+    <string name="remove_from_library">Usuń z biblioteki</string>
     <string name="mark_as_not_started">Oznacz jako nie rozpoczęty</string>
     <string name="mark_as_completed">Oznacz jako ukończony</string>
-    <string name="items_selected">%d zaznaczonych</string>
+    <plurals name="items_selected">
+        <item quantity="one">%d zaznaczona/y</item>
+        <item quantity="other">%d zaznaczone/ych</item>
```
Comment (flackbash) [inline app/src/main/res/values-pl/strings.xml:133]: Here I get an additional error message for the missing quantity *few* (and for the remaining plural strings). Again, I don't know if it makes a difference here in Polish. So if you had a good reason for leaving it out I'll merge it anyway.

**Review by flackbash — COMMENTED** [review]
Cool, thank you! I added two comments, but changes might not be necessary.

File: app/src/main/res/values-pl/strings.xml (hunk 1 above)
Comment (PatrykMis) [reply app/src/main/res/values-pl/strings.xml:31]: `other` replaces `zero` and `many` here, so `other` is enough here.

**Pull request merged**